.github/workflows/server-ci-cd.yml

```diff
@@ -1,19 +1,23 @@
 name: Server CI/CD
 on:
   push:
+    branches:
+    - master
     tags:
     - v*
 env:
-  # This is a base repository and we use ${GITHUB_REF##*/} to set the version of the container
+  # This is a base repository and we use git refs to set the version of the container
+  # ${GITHUB_REF##*/} will be either a branch name or a tag depending on the event
   REPO: aragon/govern-server
 
 jobs:
   build:
+    if: github.ref == 'refs/heads/master'
```

Comment:
I wonder if this condition is meet even on a push tag.

Reply:
Yes it is. The idea is to deploy to rinkeby on every master commit and to master when you add `v*` tag I think this makes sense?

Reply:
good point, removed this part to build on tag as well